core/src/avm2/globals/flash/xml/XMLDocument.as

```diff
@@ -1,6 +1,21 @@
 package flash.xml
 {
    public class XMLDocument extends XMLNode {
+
+      public var ignoreWhite: Boolean = false;
+
+      public function XMLDocument(input: String = null) {
+         super(0, null);
+         _isDocument = true;
+         parseXML(input);
+      }
+
+      public function parseXML(input: String): void {
+         // Unlike the new AS3 XML class, XMLDocument allows parsing
+         // multiple elements like <a>1</a><b>2</b>. So introduce a wrapper.
+         _xml = new XML("<xml>" + input + "</xml>");
```

Comment:
I think this should set `XML.ignoreWhite = this.ignoreWhite`- eventually Ruffle will implement it.